lib/unique_elements.rb

```diff
@@ -1,6 +1,21 @@
 # This method takes an array of elements (these might be strings, integers, floats, or a
 # combination of the above), and returns an array of the elements that appear once and only
 # once.
-def find_unique_elements(arr)
+
+def count_elements(arr)
+  h={}
+  arr.each do |n|
+    if h.has_key?(n) == false
+      h[n]=1
+    elsif h.has_key?(n) == true
+      h[n]= h[n] +1
+    end
+  end
```

Comment:
One definite problem is that you've gobbled up an `end` – so your `count_elements` method is never ended. This will cause your code to error out, so that should be fixed first.